### PR TITLE
Translate appendix to German

### PR DESCRIPTION
**Pull request by scy**
I’ve translated the appendix to German. Some remarks:

* I’m using “Kindesmissbrauchsabbildungen” (“depictions of child abuse”) instead of “Kinderpornografie” (“child pornography”). There is significant consensus among German speakers that “pornography” should be reserved to consensually created media and thus “child pornography” is an obsolete term.
* I’m using “Transfeindlichkeit” (“trans hostility”) instead of “Transphobie” (“transphobia”). There is broad consensus in the German-speaking trans community that calling it a phobia is misleadingly granting “transphobes” an excuse (as in “oh, it’s an illness, I’m innocent for hating trans people”).
* I’ve translated the quotes as well.

~~This is marked as a draft since the original translator of the German main page, @mrdrogdrog, has offered to proofread and review my translation.~~ **Edit:** Reviewed. (Thanks everyone!)

### CODE REVIEW

File: appendix.de.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+[9]: https://web.archive.org/web/20210325014249/https://stallman.org/archives/2012-nov-feb.html#04_January_2013_(Pedophilia)
+[10]: https://web.archive.org/web/20210325015259/https://stallman.org/archives/2019-jul-oct.html#14_September_2019_(Sex_between_an_adult_and_a_child_is_wrong)
+
+2015 und 2016 schrieb RMS drei Beiträge über das Down-Syndrom auf seiner Website. Er schlug vor, dass, sollten Personen herausfinden, dass sie schwanger sind und das Kind positiv auf das Down-Syndrom getestet wird, „die richtige Vorgehensweise für die Frau darin besteht, die Schwangerschaft abzubrechen.“[11] Er bezeichnete Menschen, die sich entscheiden, „Föten mit Down-Syndrom auszutragen“ als „pervers“ und sagte, dass „nichts tugendhaftes“ daran sei, „die Anzahl der Menschen mit Down-Syndrom [zu erhöhen].“[12] Er sagte auch „wenn ein Fötus das Down-Syndrom hat, sollte man ihn abtreiben und es nochmal probieren.“[13] In mindestens einem Fall verglich RMS ein Kind mit Down-Syndrom zu haben mit einem Haustier.[14] (Hinweis: Link 14 am 25. März 2021, 11:33 EDT hinzugefügt.)
```
Comment (mrdrogdrog) [inline appendix.de.md:24]:
```suggestion
2015 und 2016 schrieb RMS drei Beiträge über das Down-Syndrom auf seiner Website. Er schlug vor, dass, falls Personen herausfinden, dass sie schwanger sind und das Kind positiv auf das Down-Syndrom getestet wird, „die richtige Vorgehensweise für die Frau darin besteht, die Schwangerschaft abzubrechen.“[11] Er bezeichnete Menschen, die sich entscheiden, „Föten mit Down-Syndrom auszutragen“ als „pervers“ und sagte, dass „nichts tugendhaftes“ daran sei, „die Anzahl der Menschen mit Down-Syndrom [zu erhöhen].“[12] Er sagte auch „wenn ein Fötus das Down-Syndrom hat, sollte man ihn abtreiben und es nochmal probieren.“[13] In mindestens einem Fall verglich RMS ein Kind mit Down-Syndrom zu haben mit einem Haustier.[14] (Hinweis: Link 14 am 25. März 2021, 11:33 EDT hinzugefügt.)
```

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]
optional

File: appendix.de.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+[12]: https://web.archive.org/web/20210325014343/https://stallman.org/archives/2015-jul-oct.html#21_October_2015_(Mistaking_a_fetus_for_a_baby)
+[13]: https://web.archive.org/web/20210325014628/https://stallman.org/archives/2016-mar-jun.html#23_April_2016_(Fetuses_with_Downs_syndrome)
+[14]: https://web.archive.org/web/20161107050933/https://stallman.org/archives/2016-jul-oct.html#31_October_2016_(Down's_syndrome)
+
+RMS hat Jahre mit einer Kampagne gegen die Verwendung der korrekten Pronomen von Menschen verbracht. Das ist schlecht verkleidete Transfeindlichkeit. In der ursprünglichen Veröffentlichung der GNU-Richtlinien für Freundliche Kommunikation schrieb er „es gibt verschiedene Wege, Geschlechtsneutralität bei Pronomen der dritten Person Singular im Englischen auszudrücken; man muss nicht ‚they‘ benutzen.“[15] Dieser Text wurde inzwischen aktualisiert, ist aber immer noch transfeindlich.[16] Die Hauptseite seiner Website beinhaltet die Aussage „‚They‘ ist ein Plural – für Bezugsworte im Singular sollten geschlechtsneutrale Pronomen im Singular verwendet werden.“[17]
```
Comment (bhaak) [inline appendix.de.md:31]: Statt "schlecht verkleidete Transfeindlichkeit" eher "schlecht getarnte Transfeindlichkeit".
Das "schlecht" ist auch etwas holprig, vielleicht am besten "dürftig getarnte Transfeindlichkeit".

**Comment by janl** [comment]
+1 for the original text plus suggestions from another native speaker ✌️

**Review by mrdrogdrog — APPROVED** [review]

**Pull request merged**